doc/changes/latest.inc

```diff
@@ -23,7 +23,7 @@ Enhancements
 ~~~~~~~~~~~~
 .. - Add something cool (:gh:`9192` **by new contributor** |New Contributor|_)
 
-- Nothing yet
+- New function :func:`mne.chpi.get_chpi_info` to retrieve basic information about the cHPI system used when recording MEG data (this functionality was hidden in a private function before) (:gh:`9369` by `Richard Höchenberger`_)
```

Comment:
Users shouldn't ever need to care or think about or private functions (I would say it's better if they don't know of their existence, lest they get curious about what other functionality we're hiding :) )
```suggestion
- New function :func:`mne.chpi.get_chpi_info` to retrieve basic information about the cHPI system used when recording MEG data (:gh:`9369` by `Richard Höchenberger`_)
```